classes/basement/rootrecipe.yaml

```diff
@@ -44,6 +44,9 @@ depends:
           - name: devel::automake
             use: [tools]
             forward: True
+          - name: devel::texinfo
```

Comment:
Could you add it to the list of available tools in `README.md` and `doc/usage.rst`?